### PR TITLE
Add warning for unpublishing owned records

### PR DESCRIPTION
**Pull request by unclecheese**
Resolves https://github.com/silverstripe/silverstripe-versioned/issues/26

### CODE REVIEW

File: client/lang/src/en.json [hunk 1]
```diff
@@ -31,5 +31,8 @@
     "Admin.VALIDATOR_MESSAGE_DATE": "{name} is not a proper date format.",
     "Admin.VALIDATOR_MESSAGE_ALPHANUMERIC": "{name} is not an alphanumeric value.",
     "Admin.VALIDATOR_MESSAGE_ALPHA": "{name} is not only letters.",
-    "Admin.VALIDATOR_MESSAGE_DEFAULT": "{name} is not a valid value."
```
Comment (tractorcow) [inline client/lang/src/en.json]: Can we update the `client/lang/en.js` as well?

File: client/src/legacy/LeftAndMain.EditForm.js [hunk 2]
```diff
@@ -433,6 +433,39 @@ $.entwine('ss', function($){
...
+  // Hack: silverstripe/versioned coupling
+  $('.cms-edit-form .btn-toolbar button[name=action_doUnpublish].btn.action').entwine({
+    onclick: function(e) {
+      var owners = this.data('owners');
+      if (owners && parseInt(owners) > 0) {
+        var message = [
+          i18n.inject(
+            i18n._t(
+              'Admin.OWNED_WARNING_1',
+              'You are unpublishing content that is being used in {count} other published section(s).',
```
Comment (tractorcow) [inline client/src/legacy/LeftAndMain.EditForm.js]: Trailing `,` is out of place here I think.

Comment (flamerohr) [reply client/src/legacy/LeftAndMain.EditForm.js]: it's a valid trailing comma, in fact a future eslint rule ENFORCES trailing commas for multi-line params

Comment (tractorcow) [reply client/src/legacy/LeftAndMain.EditForm.js]: So weird.

**Pull request merged**